data/domain/player.tsx

```diff
@@ -708,11 +738,86 @@ export const updatePlayerDeathnote = (data: Map<string, any>) => {
     })
 }
 
+export const updatePlayerTalentLevelESBonus = (data: Map<string, any>) => {
+    const players = data.get("players") as Player[];
+    const family = data.get("family") as Family;
+
+    // Max talent level from Elemental Sorcerer
+    players.forEach(player => {
+        const esBonus = Math.floor(family.classBonus.get(ClassIndex.Elemental_Sorcerer)?.getBonus(player) ?? 0);
+        // Only update if different.
+        if (player.extraLevelsFromES == 0 || player.extraLevelsFromES != esBonus) {
+            player.talents.filter(talent => ![149, 374, 539, 505].includes(talent.skillIndex) && talent.skillIndex <= 614 && !(49 <= talent.skillIndex && 59 >= talent.skillIndex))
+                .forEach(talent => {
+                    talent.level += talent.level > 0 ? esBonus : 0;
+                    talent.maxLevel += esBonus;
+                });
+            player.extraLevelsFromES = esBonus;
+        }
+    })
+
+    return players;
+}
+
+export const updatePlayerTalentLevelExceptESBonus = (data: Map<string, any>) => {
+    const players = data.get("players") as Player[];
+    const divinity = data.get("divinity") as Divinity;
+    const equinox = data.get("equinox") as Equinox;
+    const achievementsInfo = data.get("achievements") as Achievement[];
+
+    // Update max talent level due to bear.
+    // Update players talents levels due to elite class level increase talents.
```

Comment:
You copied my comment maybe but if I'm not wrong it's only bear here? I don't remember where the symbols of beyond stuff comes from 😆

Reply:
Yeah indeed x)
Symbols of beyond are the talents, it's done in the parseTalents function

Removed the unnecessary comment line